src/routes/_components/profile/AccountProfile.html

```diff
@@ -4,7 +4,9 @@ <h1 class="sr-only">Profile for {accountName}</h1>
   <div class="account-profile-grid-wrapper">
     <div class="account-profile-grid">
       <AccountProfileHeader {account} {relationship} {verifyCredentials} />
+      {#if $isUserLoggedIn}
       <AccountProfileFollow {account} {relationship} {verifyCredentials} />
+      {/if}
```

Comment:
So users can browse other users' profiles even if they're not logged in?

Reply:
Exactly :100: